pallets/exchange/src/lib.rs

```diff
@@ -207,6 +207,30 @@ pub mod pallet {
 	pub type ExchangeAssetsIntentions<T: Config> =
 		StorageMap<_, Blake2_128Concat, (AssetId, AssetId), Vec<Intention<T>>, ValueQuery>;
 
+	#[allow(dead_code)]
```

Comment:
Do we need this here if we don't use it anywhere?

Reply:
clippy complains that the functions we have added are not used anywhere.

Reply:
do we need the functions tho?

Reply:
Ah is this what creates the constants? We should add it to xyk (since it's limit mostly of xyk not exchange) too if yes. Or maybe I'm too fast and missed something.

Reply:
I would say yes because the task was to make AMM constants( min_trading_limit, min_pool_liquidity, max_in_ratio, max_out_ratio ) accessible from exchange public API.

Reply:
The constants are also accessible from xyk and lbp pallet.

Reply:
It makes sense from the UI standpoint to have it here but it might be the case that different AMM implementations have different limits. What do we do in that case? Use the strict one for everything? We'll need to document this properly in such case.

Reply:
Yes, you are right. Different AMMs use different values. But exchange pallet uses only one pool, specified in Config::AMMPool. So we always get what we expect, the values used by exchange pallet.

Reply:
Ok, we'll want it to use multiple pools later but that will probably be task for future then.